xcell/tests/test_mapper_CatWISE.py

```diff
@@ -0,0 +1,88 @@
+import numpy as np
+import xcell as xc
+import healpy as hp
+import os
+from astropy.table import Table
+
+
```

Comment:
general question about the tests: are you ever testing the lines of code that mask the circles around the sources? For some reason coveralls isn't really complaining, but I can't see where that's happening...

Reply:
No, I didn't write a test for it. In the test for get_mask I've used `config.pop('mask_sources')` so neglecting the holes. Should I include a test for it?

Reply:
ok, let's move on without this test and circle back if we need it.